lib/ansible/modules/network/nxos/nxos_install_os.py

```diff
@@ -35,13 +35,9 @@
       - N9k 7.0(3)I4(6), 7.0(3)I5(3), 7.0(3)I6(1), 7.0(3)I7(1), 7.0(3)F2(2), 7.0(3)F3(2)
       - N3k 6.0(2)A8(6), 6.0(2)A8(8), 7.0(3)I6(1), 7.0(3)I7(1)
       - N7k 7.3(0)D1(1), 8.0(1), 8.2(1)
-    - This module executes longer then the default ansible timeout value and
-      will generate errors unless the module timeout parameter is set to a
-      value of 500 seconds or higher.
-      The example time is sufficent for most upgrades but this can be
-      tuned higher based on specific upgrade time requirements.
-      The module will exit with a failure message if the timer is
-      not set to 500 seconds or higher.
+    - This module requires both the ANSIBLE_PERSISTENT_CONNECT and
+      ANSIBLE_PERSISTENT_COMMAND timers to be set to 600 seconds or higher.
```

Comment:
Can you please change the env vars to `ANSIBLE_PERSISTENT_COMMAND_TIMEOUT` and `ANSIBLE_PERSISTENT_CONNECT_TIMEOUT` instead?